book/src/integrations/oauth2.md

```diff
@@ -130,13 +130,22 @@ After you have understood your service requirements you first need to configure
 members of `system_admins` or `idm_hp_oauth2_manage_priv` are able to create or manage OAuth2 client
 integrations.
 
-You can create a new client with:
+You can create a new client by specifying it's client name, application display name and the landing
+page (home page) of the client. The landing page is where users will be redirect to in order to
+begin an oauth2 flow.
```

Comment:
Is that only used for the kanidm's dashboard then? Because generally I'd expect the client to initiate the flow. If it's the case, I suggest rephrasing this to point this exact usage.

Reply:
Clarified.